src/frontend/components/angular-mocks/bower.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "angular-mocks",
+  "version": "1.3.20",
```

Comment:
It seems that `angular-mocks` can be installed via npm as well: https://www.npmjs.com/package/angular-mocks